tests/end-to-end/external-solution-integration/pkg/testsuite/send_event_to_mesh_and_check_eventId.go

```diff
@@ -0,0 +1,107 @@
+package testsuite
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+
+	retrygo "github.com/avast/retry-go"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/internal/example_schema"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
+	"github.com/sirupsen/logrus"
+)
+
+// SendEventToMeshAndCheckEventId is a step which sends an event and checks if the correct EventId has been received
+type SendEventToMeshAndCheckEventId struct {
+	state       SendEventState
+	appName     string
+	payload     string
+	testService *testkit.TestService
+	retryOpts   []retrygo.Option
+}
+
+var counterMesh = 0
```

Comment:
Does it make sense to move this inside the `SendEventToMeshAndCheckEventId` struct definition?

Reply:
Fixed